### DISCUSSION TITLE
implementation IOB/last bolus/basal active/last cho

### DISCUSSION THREAD

**Discussion by painbrain81** [kashamalasha/nightscout-widget-electron#74] [Ideas]
Hi!
it would be useful for the user to have (in small print under the blood sugar) information about the parameters in question (available on NS).
It would be nice if they could be activated/deactivated from the preferences window.

**Comment by painbrain81** [comment]
like this.
more chaotic but more information-full!

![2023-11-23_085255](https://github.com/kashamalasha/nightscout-widget-electron/assets/24631990/bcd1e265-7b5c-41d8-b944-1bc86f5e0878)

**Reply by kashamalasha** [reply]
Hi! Thanks for the help. It looks a bit overwhelming. I need to think about a nice design for this data.
The main point of this app was its minimalistic appearance. If the user needs to check all necessary parameters on the Nightscout, he/she/they could go to their Nightscout by clicking the graph button.
